docs/source/markdown/podman-machine-set.1.md

```diff
@@ -68,6 +78,11 @@ To switch the VM `myvm` from rootless to rootful:
 $ podman machine set --rootful myvm
 ```
 
+To add more two additional disks of 50GiB to the VM `myvm`:
+```
+$ podman machine set --extra-disk-num 2 --extra-disk-size 50 myvm
```

Comment:
Just verifying, and I haven't seen below yet, but is it really `GiB` (1024 to the 3rd) versus `GB` (1000 to the 3rd)?

Reply:
Good point, it's actually GB, not GiB. I'll change that. Thanks!